xbmc/input/KeyboardStat.cpp

```diff
@@ -32,6 +32,8 @@
 #include "threads/SystemClock.h"
 #include "utils/log.h"
 
+#define HOLD_THRESHOLD 250
```

Comment:
while working on RetroPlayer, I had a user request a hold duration, e.g. <start mod="longpress" duration="1000">ActivateWindow(Home)</guide>

is hold duration something that could be easily added later?